.github/workflows/publish-dev_gen.yaml

```diff
@@ -91,36 +93,13 @@ jobs:
       run: |-
         poetry config repositories.featurebyte_np https://${GCR_PYPI_LOCATION}-python.pkg.dev/${GCR_PYPI_PROJECT_ID}/${GCR_PYPI_REPO}
         poetry publish --build -r featurebyte_np --username='_json_key_base64' --password='${{ secrets.GCR_PYPI_CREDENTIALS_RW_B64 }}'
-#    - name: Setup Docker
-#      uses: docker/setup-buildx-action@v2
-#    - name: Docker Login to Non-Prod Artifact Registry
-#      uses: docker/login-action@v2
-#      with:
-#        password: ${{ secrets.GCR_DOCKER_CREDENTIALS_RW_B64 }}
-#        registry: ${{ env.GCR_DOCKER_LOCATION }}-docker.pkg.dev
-#        username: _json_key_base64
-#    - name: Build & Tag & Push Image to NonProd
-#      uses: docker/build-push-action@v4
-#      with:
-#        push: true
-#        tags: |-
-#          ${{ env.SERVER_IMAGE_REPO }}:${{ env.SHORT_HASH }}
-#          ${{ env.SERVER_IMAGE_REPO }}:${{ env.DEV_VERSION }}
-#          ${{ env.SERVER_IMAGE_REPO }}:latest
-#        context: .
-#        platforms: |-
-#          linux/amd64
-#          linux/arm64
-#        file: docker/Dockerfile
-#        cache-from: type=registry,ref=${{ env.SERVER_IMAGE_REPO }}:latest
-#        cache-to: type=inline
```

Comment:
This one doesnt matter. It will only run on pushes to main branch.
Essentially this workflow is a NOP when merged to release branch